requirements.txt

```diff
@@ -78,7 +79,7 @@ zope.interface==4.3.2
 defusedxml==0.6.0
 M2Crypto
 python-memcached==1.48
--e git+https://github.com/GSA/pysaml2.git@datagov/v4.9.0#egg=pysaml2
+-e git+https://github.com/GSA/pysaml2.git@c26a4893ff4c8d97e5f7994809ccb96a9d12caa3#egg=pysaml2
```

Comment:
Shouldn't this be a branch name, so that the update process picks up new commits?

Reply:
Good question. I got the hash from catalog deployment, but I will take a close look to see if we can use branch.